resources/views/users/_user_info.blade.php

```diff
@@ -10,6 +10,11 @@
     @endcan
 
     <h2>{{ $user->name() }}</h2>
+    @if ($user->hasBio())
```

Comment:
Replace this with `@if ($bio = $user->bio())` and use the `$bio` variable below.